09-valve-monitor/firmware/platformio.ini

```diff
@@ -13,7 +13,8 @@ platform = ststm32
 board = bw_swan_r5
 build_flags = 
 	-D PIO_FRAMEWORK_ARDUINO_ENABLE_CDC
-upload_protocol = stlink
+	-D USE_VALVE=1
+upload_protocol = dfu
```

Comment:
We’ve used stlink for all the other `platformio.ini` files, and the README also assumes this is set to stlink https://github.com/blues/app-accelerators/tree/main/09-valve-monitor/firmware#flashing

Reply:
updated